src/Emulator.cpp

```diff
@@ -61,7 +61,7 @@ namespace sn
         m_ppu.reset();
 
         m_window.create(sf::VideoMode(NESVideoWidth * m_screenScale, NESVideoHeight * m_screenScale),
-                        "SimpleNES", sf::Style::Titlebar | sf::Style::Close);
+                        "SimpleNES", sf::Style::Titlebar | sf::Style::Close | sf::Style::Resize);
```

Comment:
Adding resize also allows the user to set the window to incorrect aspect ratio. That's why I initially didn't allow it and instead supported a command line argument to resize. But this is okay for now.

Reply:
sorry i read only now, thanks :) i know that aladdin if i remember correctly uses illegal opcodes.